lib/Agrammon/Web/Routes.pm6

```diff
@@ -216,7 +216,7 @@ sub api-routes (Str $schema, $ws) {
         operation 'storeData', -> LoggedIn $user {
             request-body -> ( :datasetName($dataset-name), :$variable, :$value, :@branches, :@options , :$row) {
                 $ws.store-data(
-                    $user, $dataset-name, $variable, $value, @branches, @options, $row
+                    $user, $dataset-name, $variable, $value, @branches, @options, $row // Any
```

Comment:
@jnthn Is `// Any` the right thing to do here?

Reply:
As discussed, better `Int :$row` or similar in the destrcuture.

Reply:
Done.